### PR TITLE
Improve regexes for ssh configuration

### PR DESCRIPTION
**Pull request by dblenkus**

### CODE REVIEW

File: tasks/security.yml [hunk 1]
```diff
@@ -14,15 +14,15 @@
 - name: Disallow password authentication
   lineinfile:
     dest: /etc/ssh/sshd_config
-    regexp: "PasswordAuthentication"
+    regexp: "^(#\s*)?PasswordAuthentication\s+(yes|no)"
```
Comment (tjanez) [inline tasks/security.yml]: You need to escape `\` with another `\`. The regexp should be:
```
"^(#\\s*)?PasswordAuthentication\\s+(yes|no)"
```

File: tasks/security.yml [hunk 2]
```diff
@@ -14,15 +14,15 @@
...
     line: "PasswordAuthentication no"
     state: present
   notify: Restart ssh
 
 - name: Disallow root login
   lineinfile:
     dest: /etc/ssh/sshd_config
-    regexp: "PermitRootLogin"
+    regexp: "^(#\s*)?PermitRootLogin\s+(yes|no)"
```
Comment (tjanez) [inline tasks/security.yml]: Same here.

**Review by tjanez — CHANGES_REQUESTED** [review]
Otherwise, looks good.

I've tested it on CentOS 7.2 and RHEL 7.2 and it works.

**Comment by dblenkus** [comment]
Fixed

**Review by tjanez — APPROVED** [review]

**Pull request merged**